interfaces/Portalicious/src/app/models/program.model.ts

```diff
@@ -0,0 +1,7 @@
+import { ProgramStats } from '@121-service/src/metrics/dto/program-stats.dto';
+import { ProgramController } from '@121-service/src/programs/programs.controller';
+import { Dto121Service } from '~/utils/dto-type';
+
+export type Program = Dto121Service<ProgramController['findOne']>;
+
+export type ProgramMetrics = ProgramStats;
```

Comment:
I've tried to import it from the `MetricsController` (see commit);

But it triggered all kinds of type-errors and complaints about code in the 121-service. :'(

Reply:
With this solution there are still changes required in the `121-service`, but I think they're not super-big... (But I haven't really tested them)